completions/reportbug

```diff
@@ -27,41 +27,19 @@ _reportbug()
                 done)' -- "$cur"))
             return
             ;;
-        --bts | -!(-*)B)
-            COMPREPLY=($(compgen -W "debian guug kde mandrake help" -- \
-                "$cur"))
+        --tag | --ui | --interface | --type | --bts | --severity | --mode | -!(-*)[TutBS])
```

Comment:
For example, like this

```suggestion
        --tag | --ui | --interface | --type | --bts | --severity | --mode | -!(-*|*[CefKHPsoiATjVuQtBS]*)[TutBS])
```

Hmm, but there seem to be many similar `case` patterns for other options. Maybe we do not have to care about it too much.

Reply:
I now think we should exclude the patterns of an option flag plus an option argument. I also think we should update all the similar patterns for e.g. the following case:

```
$ reportbug -tdebbugs -[TAB]
```

This should complete into a list of possible options, but I think the current completion function tries to complete it to a list of possible option arguments for `-s` because `-tdebbugs` matches `-!(-*)s` and fails.

Reply:
I'm torn on this. While I agree it would be the right thing to do, we have a lot of affected completions throughout the tree. If we are to fix them, it would be good to do it in a consistent manner (for example to use the same variable name to hold appropriate flags and reuse it for all affected arg completion cases within the file), and even more so to take care of them ~everywhere instead of just singling it out here. Also, taking care of this would be quite a bit of work for a relatively rare use case/bug.

Reply:
I see. I agree that we can do it in a consistent manner for the whole tree, but the actual problem here is that the following hangs:

```bash
compgen -W '$(reportbug -utext help 2>&1 | sed /^[[:space:]]/p)'
```

because `reportbug` tries to start an interactive session and wait for the user input. I think an alternative workaround is to redirect stdin to `/dev/null` so `reportbug` gives up starting the interactive session.

```bash
compgen -W '$(reportbug -utext help 2>&1 </dev/null | sed /^[[:space:]]/p)'
```

Reply:
Addressed below in 3f9970daf5c84878cd67576268888cd7d82d14fb

Reply:
Thanks!